Source/Mosa.DeviceDriver/ISA/PIT.cs

```diff
@@ -60,12 +60,12 @@ public class PIT : DeviceSystem.DeviceDriver
 		/// <summary>
 		/// Initializes this device.
 		/// </summary>
-		protected override void Initialize()
+		public override void Initialize()
```

Comment:
Public?

Reply:
Yeah, BaseDeviceDriver requires that function to be public.

Reply:
It didn't before. What changed?

Reply:
Never mind!